beta-src/src/state/game/game-api-slice.ts

```diff
@@ -391,6 +509,43 @@ const gameApiSlice = createSlice({
             }
           }
           state.order.type = "hold";
+        } else if (
+          order.onTerritory !== null &&
+          Territory[order.onTerritory] === territoryName &&
+          !order.type &&
+          phase === "Retreats"
+        ) {
+          let command: GameCommand = {
+            command: "DISBAND",
+          };
+
+          setCommand(state, command, "territoryCommands", territoryName);
+          setCommand(state, command, "unitCommands", currOrderUnitID);
+          command = {
+            command: "REMOVE_ARROW",
+            data: {
+              orderID: order.orderID,
+            },
+          };
+          setCommand(state, command, "mapCommands", "all");
+          if (currentOrders) {
+            const orderToUpdate = currentOrders.find((o) => {
+              return o.unitID === currOrderUnitID;
+            });
+            console.log("otu", orderToUpdate);
+            if (orderToUpdate) {
+              updateOrdersMeta(state, {
+                [orderToUpdate.id]: {
+                  saved: false,
+                  update: {
+                    type: "Disband",
+                    toTerrID: null,
+                  },
+                },
+              });
+            }
+          }
```

Comment:
Unnecessary if statement. If an order is in progress, you should have the orderID.

Reply:
I had to change the interface export to export default or else it read it as possibly undefined